clients/spark/core/src/main/hadoop2/scala/io/treeverse/clients/conditional/S3ClientBuilder.scala

```diff
@@ -28,16 +30,49 @@ object S3ClientBuilder extends io.treeverse.clients.S3ClientBuilder {
         )
       } else None
 
+    initializeClient(configuration, credentialsProvider, region, bucket)
+  }
+
+  private def initializeClient(
+      config: ClientConfiguration,
```

Comment:
Let's keep a single name, `configuration` or `config`: we read code more than we type it, and reading this function I wondered whether it was the same "configuration" from above.